### DISCUSSION TITLE
posts/wzjdccsdgj/

### DISCUSSION THREAD

**Discussion by giscus** [Shitao5/shitao-blog#53] [Comments]
# posts/wzjdccsdgj/

对认知的认知，叫元认知。制作工具的工具，姑且叫「元工具」，比如机床。
[&hellip;] 这里有可以无限套娃的牛角尖：机床是制作工具的工具，那制作机床也需要工具，那到底谁是真正的「元工具」？这个问题似曾相识吧！没错，鸡生蛋还是蛋生鸡，在这个宏大问题面前投降，但采取个体分析是一个思路：这只鸡生了这个蛋，这个蛋孵化了那只鸡。
[&hellip;] 工具达成两个目标：让不可能的事情发生（一个人靠千斤顶 &hellip;

https://shitao5.org/posts/wzjdccsdgj/

**Comment by yihui** [comment]
纠正两个字符：c -> z（除非你家的方言和我家一样把“造”读 cao）、论 -> 轮。

**Reply by yihui** [reply]
还要纠正 ɡ -> g，我都不知道怎样才能打出[前一个字符](https://zh.wikipedia.org/wiki/%E6%BF%81%E8%BB%9F%E8%85%AD%E5%A1%9E%E9%9F%B3)来。

**Reply by Shitao5** [reply]
谢谢 yihui，俺在用 ubuntu 自带的中文输入法，确实还不太适应，容易打错字，居然还没发现。

> 除非你家的方言和我家一样把“造”读 cao

文件名和 slug 是用 [pinyin](https://github.com/pzhaonet/pinyin) 包转过来的，看了包里的数据：[`造 cāo cào zào(cao1 cao4 zao4)`](https://github.com/pzhaonet/pinyin/blob/master/inst/lib/pinyin.txt#L22646)，莫非是湖北的大佬整理的。git 提交的时候发现 g 不大对劲，前一个字符连复制都复制不出来……改了文件名忘改 slug 了。试了下加一个不要音标的参数就没问题了。

![图片](https://github.com/user-attachments/assets/aa204d5f-7911-4087-93ac-dd84f8975ef7)

俺家方言“造”是读 zao，俺也是第一次听说有读 cao 的。搜了搜[湖北口音](https://m.thepaper.cn/baijiahao_7511773)，学习曲线陡峭，slug 留着 c 当俺第一次学习了，下次去当地感受下哈哈。

**Reply by CyrusYip** [reply]
@Shitao5 https://github.com/Shitao5/shitao-blog/discussions/53#discussion-7012282

这里底部的链接还没改。

---

Ubuntu 默认的输入法是 ibus 吗？我在 Linux 一直用 Fcitx5，搭配[维基百科词典](https://github.com/felixonmars/fcitx5-pinyin-zhwiki)和云拼音，基本不会打错字。我现在因为保护隐私不用云拼音，只用维基百科词典也够了。

一些可能帮到你的参考资料：

- [在 Ubuntu 下配置 Fcitx5 | 叶寻的博客](https://cyrusyip.org/zh-cn/posts/2020/11/06/configure-fcitx5-on-ubuntu/) （装词典推荐放到本地目录，ArchWiki 有写）
- [Fcitx5 - ArchWiki](https://wiki.archlinux.org/title/Fcitx5)

**Reply by Shitao5** [reply]
@CyrusYip  谢谢叶寻，在 Linux 操作上你每次都给我很多帮助。Ubuntu 默认是 ibus，我也是基于隐私的考虑没有使用云拼音。

Fcitx5 我在刚装上 Ubuntu 24.04 的时候安装了，当时好像是有个库过时了还是咋没处理好就耽搁了，一直在用默认的中文输入法。最近用下来发现默认输入法属实不好使，还是抽空折腾下装上趁手的工具吧哈哈。
